src/message.rs

```diff
@@ -58,7 +85,42 @@ impl fmt::Display for Message {
             self.location.column(),
             ":".cyan(),
             self.kind.code().as_ref().red().bold(),
-            self.kind.body()
-        )
+            self.kind.body(),
+        );
+        let slices = if self.show_source && self.source.is_some() && self.range.is_some() {
+            vec![Slice {
+                source: self.source.as_ref().unwrap(),
```

Comment:
Nit: maybe `if let` to avoid some of the unwraps?

Reply:
Sounds good. Will fix.